core/master/src/main/resources/config/commands/ftp/core.conf

```diff
@@ -410,21 +410,60 @@ help Add a new user
 help.specific ${command} <user> <password> [<ident@ip#1> ...]
 }
 
+SITE ADDGROUP {
+plugin org.drftpd.commands.usermanagement
+class GroupManagementHandler
+method doSITE_ADDGROUP
+perms =siteop
+help Add a new group
+help.specific ${command} <group>
+}
+
+# Backwards compatibility
 SITE CHANGE {
```

Comment:
Why keeping SITE CHANGE? Looks like the same as SITE CHANGEUSER

Reply:
This was a request by harrox for backwards compatability for various scripts in ftp clients.